lunatik.h

```diff
@@ -322,10 +322,15 @@ do {									\
 	lua_pop(L, 1);							\
 } while (0)
 
+static inline void lunatik_checkbounds(lua_State *L, int idx, lua_Integer val, lua_Integer min, lua_Integer max)
+{
+	luaL_argcheck(L, val > min && val <= max, idx, "out of bounds");
+}
```

Comment:
I think in that case, the inline is a bit too much.. I would just use a macro here..
```suggestion
#lunatik_checkbounds(L, idx, val, min, max)	luaL_argcheck(L, val >= min && val <= max, idx, "out of bounds")
```
I would use `>=` to be consistent with `<=`, so, both boundaries are inclusive.. then, we can pass `1` instead of `0` on the call..